src/PointwiseFunctions/AnalyticData/CMakeLists.txt

```diff
@@ -2,5 +2,6 @@
 # See LICENSE.txt for details.
```

Comment:
remove the caveat from the commit message as we do not support non-self starting time steppers any more.

Reply:
removed